.gitignore

```diff
@@ -0,0 +1,61 @@
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+
+# Runtime data
+pids
+*.pid
+*.seed
+*.pid.lock
+
+# Directory for instrumented libs generated by jscoverage/JSCover
+lib-cov
+
+# Coverage directory used by tools like istanbul
+coverage
+
+# nyc test coverage
+.nyc_output
+
+# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
+.grunt
+
+# Bower dependency directory (https://bower.io/)
+bower_components
```

Comment:
Are we planning on using Grunt and Bower?

Reply:
not particularly, no.

Reply:
Should we just drop these lines then?

Reply:
🤷 copy pasted. didn't think necessary to go through and scrub each line. if we want to get real picky, we can start from scratch and not grab from previous projects. when it comes to ignore files, i dont' care as much.